prow/overlays/cnv-prod/config.yaml

```diff
@@ -669,3 +669,99 @@ presubmits:
               requests:
                 memory: "2Gi"
                 cpu: "2"
+
+  AICoE/aicoe-cd:
+    - name: kustomize-build
+      decorate: true
+      max_concurrency: 1
+      run_if_changed: ".*yaml"
+      skip_report: false
+      context: aicoe-ci/kustomize-build
+      spec:
+        containers:
+          - image: quay.io/operate-first/opf-toolbox:v0.2.0
+            command:
+              - "./test-kustomize-build" # this is contained in the thoth-application/ repo not the opf-toolbox
```

Comment:
please remove that comment , or edit it to saying that it would be picked from `aicoe-cd` repo. 
same for the others

Reply:
fixed.